src/components/Icon/icons.embed.js

```diff
@@ -0,0 +1,101 @@
+import caretUp from '../../icons/caret-up.svg'
+import chat from '../../icons/chat.svg'
+import chatSquare from '../../icons/chat-square.svg'
+import clock from '../../icons/clock.svg'
+import clockSmall from '../../icons/clock-small.svg'
+import collapse from '../../icons/collapse.svg'
+import copy from '../../icons/copy.svg'
+import crossLarge from '../../icons/cross-large.svg'
+import crossMedium from '../../icons/cross-medium.svg'
+import crossMediumAlt from '../../icons/cross-medium-alt.svg'
+import crossSmall from '../../icons/cross-small.svg'
+import document from '../../icons/document.svg'
+import emoji from '../../icons/emoji.svg'
+import ellipsis from '../../icons/ellipsis.svg'
+import fabAntenna from '../../icons/fab-antenna.svg'
+import fabBuoy from '../../icons/fab-buoy.svg'
+import fabChat from '../../icons/fab-chat.svg'
+import fabQuestion from '../../icons/fab-question.svg'
+import fabSearch from '../../icons/fab-search.svg'
+import fullscreen from '../../icons/fullscreen.svg'
+import helpscoutLogo from '../../icons/helpscout-logo.svg'
+import hsLogo from '../../icons/hs-logo.svg'
+import image from '../../icons/image.svg'
+import imageAdd from '../../icons/image-add.svg'
+import imageLarge from '../../icons/image-large.svg'
+import info from '../../icons/info.svg'
+import link from '../../icons/link.svg'
+import markup from '../../icons/markup.svg'
+import meatball from '../../icons/meatball.svg'
+import minimize from '../../icons/minimize.svg'
+import note from '../../icons/note.svg'
+import plusLarge from '../../icons/plus-large.svg'
+import search from '../../icons/search.svg'
+import spinner from '../../icons/spinner.svg'
+import text from '../../icons/text.svg'
+import tickLarge from '../../icons/tick-large.svg'
+import tickSmall from '../../icons/tick-small.svg'
+import trash from '../../icons/trash.svg'
+import user from '../../icons/user.svg'
+
+const ICONS = {
+  alert,
+  'arrow-left': arrowLeft,
+  'arrow-right': arrowRight,
+  attachment,
+  bin: trash,
+  'caret-down': caretDown,
+  'caret-left': caretLeft,
+  'caret-right': caretRight,
+  'caret-up': caretUp,
+  chat: chatSquare,
+  'chat-square': chatSquare,
+  'chat-round': chat,
+  'clock-small': clockSmall,
+  clock,
+  collapse,
+  copy,
+  'cross-large': crossLarge,
+  'cross-medium-alt': crossMediumAlt,
+  'cross-medium': crossMedium,
+  'cross-small': crossSmall,
+  document,
+  ellipsis,
+  emoji,
+  'fab-antenna': fabAntenna,
+  'fab-buoy': fabBuoy,
+  'fab-chat': fabChat,
+  'fab-question': fabQuestion,
+  'fab-search': fabSearch,
+  fullscreen,
+  'helpscout-logo': helpscoutLogo,
+  'hs-logo': hsLogo,
+  'image-add': imageAdd,
+  'image-large': imageLarge,
+  'image-plus': imageAdd,
+  image,
+  info,
+  information: info,
+  link,
+  markup,
+  meatball,
+  minimize,
+  note,
+  plus: plusLarge,
+  search,
+  spinner,
+  'tick-large': tickLarge,
+  'tick-small': tickSmall,
+  tick: tickLarge,
+  trash,
+  user,
+}
+
+export default ICONS
```

Comment:
This list seems to include more icons than those needed in embed. I just audited our usage of Icons in the codebase and came up with this list:

```
fullscreen
search
helpscout-logo
arrow-left image
attachment
minimize
image-add
meatball
cross-small
tick tick-small
chat
```

If we could reduce it to only these icons, the component would be a third of its current size :)

Reply:
@Charca Would you be able to double check that list? From that list, I'm sure we have (at least) the follow in Embed:

* `search`
* `helpscout-logo`
* `chat`
* `attachment`
* `image-add`
* `meatball`
* `arrow-left`
* `tick-small`

I just did a global search, and was able to find all the ☝️ from my list

Reply:
Haha yes! Sorry I wasn't clear about that!

> P.S. I need to include certain icons like alert, spinner, and cross as it comes with other components (like error UIs and Modal)

That's perfect. Thank you!